vendor/github.com/ethereum/go-ethereum/whisper/whisperv5/doc.go

```diff
@@ -99,3 +99,40 @@ type NotificationServer interface {
 	// Stop stops notification sending loop, releasing related resources
 	Stop() error
 }
+
+type envelopeSource int
```

Comment:
Is there a patch for that?

Reply:
Not yet, that's the missing part and it's denoted in the PR description.